src/poker/game/GamePlay.java

```diff
@@ -0,0 +1,545 @@
+                    e.printStackTrace();
+                }
+            }
+            for (int n = 0; n < 20; n++) {
+                System.out.println();
+            }
+            System.out.println(player1.getStackPlayer());
+            System.out.println(player2.getStackPlayer());
+            if (i % 2 == 0) {
+                upadePotStackAmountInHandBigBlindPlayer(bigBlind, player1);
+                upadePotStackAmountInHandSmallBlindPlayer(smallBlind, player2);
+                System.out.println("Zaczyna " + player2.getNamePlayer());
+                preFlopSmallBlind(player1, player2);
+                resetMaxValueRaises();
+                gameState.setGamePhase(GamePhase.FLOP);
+                outOfPositionPostFlop(player1, player2);
+                resetMaxValueRaises();
+                outOfPositionPostFlop(player1, player2);
+                resetMaxValueRaises();
+                outOfPositionPostFlop(player1, player2);
+            } else {
+                upadePotStackAmountInHandBigBlindPlayer(bigBlind, player2);
+                upadePotStackAmountInHandSmallBlindPlayer(smallBlind, player1);
+                System.out.println("Zaczyna " + player1.getNamePlayer());
+                preFlopSmallBlind(player2, player1);
+                resetMaxValueRaises();
+                gameState.setGamePhase(GamePhase.FLOP);
+                outOfPositionPostFlop(player2, player1);
+                resetMaxValueRaises();
+                outOfPositionPostFlop(player2, player1);
+                resetMaxValueRaises();
+                outOfPositionPostFlop(player2, player1);
+            }
+            firstHand = false;
+            i++;
+        }
+    }
+
+    public void updatePlayers(String player) {
+        if (player.equals("Player1")) {
+            player1.increaseStackPlyer(gameState.pot);
+            player1.decreaseAmountInHand(gameState.pot / 2);
+            player2.decreaseAmountInHand(gameState.pot / 2);
+            gameState.decreasePot(gameState.pot);
+        } else if (player.equals("Player2")) {
+            player2.increaseStackPlyer(gameState.pot);
+            player2.decreaseAmountInHand(gameState.pot / 2);
+            player1.decreaseAmountInHand(gameState.pot / 2);
+            gameState.decreasePot(gameState.pot);
+        } else {
+            player1.increaseStackPlyer(gameState.pot / 2);
+            player2.increaseStackPlyer(gameState.pot / 2);
+            player1.decreaseAmountInHand(gameState.pot / 2);
+            player2.decreaseAmountInHand(gameState.pot / 2);
+            gameState.decreasePot(gameState.pot);
+        }
+        this.result = boardAndCards.makeHandsBoard();
+        this.gameState.setGamePhase(GamePhase.PREFLOP);
+        this.isFirstRaiseBigBlind = true;
+        this.isFirstRaiseSmallBlind = true;
+        this.smallBlindCallPreFlop = true;
+    }
+
+    public void outOfPositionPostFlop(Player player1, Player player2) {
```

Comment:
niech sie to nazwya postflopAction(Player smallBlindPlayer, Player bigBlindPlayer)

Reply:
zmieniles to na postFlopActionBig(...) ten "Big" w nazwie mi sie nie podoba bo nie jest oczywiste co to oznacza imo do wywalenia albo do przeksztalcenia